doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1901,6 +1904,9 @@ Response:
 
 Only the fields included in the request are updated in the subscription.
 
+Using `expires` with value equal to empty string (`""`) equals to removing the field, i.e.
```

Comment:
On @kzangeli behalf, from the old PR:

And same here:

    Using `expires` with a value of the empty string (`""`) equals removing the field, i.e. ...